### PR TITLE
Add `ADTHAUT` to adsl

### LINKED ISSUE #199: Add `ADTHAUT` to adsl
"Autopsy Performed" variable needed in AEL04.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by edelarua**
Closes #199

### CODE REVIEW

File: vignettes/rebuild_cached_data.Rmd [hunk 1]
```diff
@@ -102,23 +102,23 @@ cadsub <- cadsub_new
...
+save(cadpc, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadpc.RData", compress = "xz")
+save(cadpp, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadpp.RData", compress = "xz")
+save(cadqs, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadqs.RData", compress = "xz")
+save(cadrs, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadrs.RData", compress = "xz")
+save(cadtr, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadtr.RData", compress = "xz")
+save(cadtte, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadtte.RData", compress = "xz")
+save(cadvs, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadvs.RData", compress = "xz")
+save(caddv, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/caddv.RData", compress = "xz")
+save(cadsub, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadsub.RData", compress = "xz")
+save(cadhy, file = "C:/Users/delaruae/NEST/random.cdisc.data/data/cadhy.RData", compress = "xz")
```
Comment (shajoezhu) [inline vignettes/rebuild_cached_data.Rmd:123]: Please revert this paths.

File: vignettes/rebuild_cached_data.Rmd [hunk 2]
```diff
@@ -11,7 +11,7 @@ vignette: >
 
 # Getting Started
 
-The following script is used to create, compare and save cached data to `data/` directory.
+The following script is used to create, compare and save cached data to `C:/Users/delaruae/NEST/random.cdisc.data/data/` directory.
```
Comment (shajoezhu) [inline vignettes/rebuild_cached_data.Rmd:14]: please revert this change

**Review by shajoezhu — CHANGES_REQUESTED** [review]
Hi @edelarua , thanks for the changes. `ADTHAUT` looks good. Seems that you have also updated some paths for the data, can you change them back please. Thanks!

**Comment by edelarua** [comment]
Hey @shajoezhu sorry about that, the paths have been reverted and it should be good to go now.

**Review by shajoezhu — APPROVED** [review]
brilliant! Thanks @edelarua

**Pull request merged**